### PR TITLE
Draft release notes for r151034

### PR DESCRIPTION
**Pull request by citrus-it**
Starter for 10, it's quite likely I've missed something in this initial pass.

@danmcd - can you please look at the NFS zone bit in particular - is there anywhere that would be worth linking to from here?

### CODE REVIEW

File: doc/ReleaseNotes.md [hunk 1]
```diff
@@ -13,29 +13,171 @@ r151034 release repository: https://pkg.omniosce.org/r151034/core
...
+https://omniosce.org/upgrade
+
 ## New features since r151032
 
 ### System Features
 
+* OmniOS now supports running an NFS server in a zone. NFS sharing can be
+  enabled by setting the `sharenfs` property on a dataset just as in the
+  global zone.
+
```
Comment (danmcd) [inline doc/ReleaseNotes.md]: Native zone.  Can't do it (easily, if at all) in LX.  Also, do any of your newer native brands do what SmartOS does and not give root@zone its own dataset/mountpoint?  If so, then you can't share NFS on such a dataset.  If you can `zfs set` in the zone, you can share it, but some old-timers might use share(1M) or sharemgr(1M) and if they share a directory on a filesystem the zone doesn't own, they'll get EPERM or something else.

File: doc/ReleaseNotes.md [hunk 2]
```diff
@@ -13,29 +13,171 @@ r151034 release repository: https://pkg.omniosce.org/r151034/core
...
+* While it was previously possible to create an SMB share within a zone via
+  the `sharemgr` command, it can now be done more easily via the `sharesmb`
+  dataset property, as in the global zone.
+
```
Comment (danmcd) [inline doc/ReleaseNotes.md]: With the caveat that all of the other things you had to do in the global zone (AD, etc. etc. if needed) ALSO need to be done in-zone.

File: doc/ReleaseNotes.md (hunk 1 above)
Comment (citrus-it) [reply doc/ReleaseNotes.md]: Thanks.
No, none of our brands do that apart from kvm/bhyve.
This should be usable in ipkg, lipkg, sparse or pkgsrc. I'll make the note more explicit.

Comment (danmcd) [reply doc/ReleaseNotes.md]: So if those zone brands have $ZONEROOT owned by root@zone, which you just said they do, you can ALSO use the share(1M)/share_nfs(1M) command on any directory in-zone for finer-grained directory access.

**Pull request merged**